src/google/protobuf/arena.cc

```diff
@@ -115,7 +115,11 @@ SerialArena::SerialArena(Block* b, void* owner) : space_allocated_(b->size) {
 SerialArena* SerialArena::New(Memory mem, void* owner) {
   GOOGLE_DCHECK_LE(kBlockHeaderSize + ThreadSafeArena::kSerialArenaSize, mem.size);
 
-  auto b = new (mem.ptr) Block{nullptr, mem.size};
+  auto b = new (mem.ptr) Block{
+      .next = nullptr,
+      .size = mem.size,
+      .start = nullptr,
+  };
```

Comment:
Designated initializers are a c++20 extension. We support c++11

Reply:
Done.